tasks/configure_namespace.yml

```diff
@@ -206,11 +206,17 @@
                     volumeMounts:
                       - name: reactor-config-map
                         mountPath: /rcm
+                      - name: remote-host-auth
+                        mountPath: "{{ zombie_slots_pruner_remote_host_mount_path }}"
```

Comment:
Could you document that the mountPath has to be the basename of all `auth` keys for all remote hosts in the configmap?

Side note: is it just me, or is the upstream role getting impossible to use without complete downstream knowledge?

Reply:
updated

well the role depends on config map ....